docs/get-started/nextjs.md

```diff
@@ -671,6 +671,106 @@ import 'styles/globals.scss';
 // ...
 ```
 
+<Callout variant="warning">
+
+Absolute imports **cannot** be mocked in stories/tests. See the [Mocking modules](#mocking-modules) section for more information.
+
+</Callout>
+
+## Module aliases
+
+[Module aliases](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases#module-aliases) are also supported.
+
+```jsx
+// index.js
+// All good!
+import Button from '@/components/button';
+// Also good!
+import styles from '@/styles/HomePage.module.css';
+
+export default function HomePage() {
+  return (
+    <>
+      <h1 className={styles.title}>Hello World</h1>
+      <Button />
+    </>
+  );
+}
+```
+
+## Subpath imports
+
+As an alternative to [module aliases](#module-aliases), you can use [subpath imports](https://nodejs.org/api/packages.html#subpath-imports) to import modules. This follows Node package standards and has benefits when [mocking modules](#mocking-modules).
```

Comment:
The term `subpath imports` is already linked to that exact URL. I could be misunderstanding your feedback. But otherwise, the fact that two of you missed that is noteworthy so I thought for a while about how to improve this... and came up empty. I really want to keep the _term_ linked, and long link text doesn't work well.

Reply:
I think I was too focused on the "This follows Node package standards" part. It's probably fine.